src/content/docs/en/reference/container-reference.mdx

```diff
@@ -40,25 +40,30 @@ Enables rendering components using [HTML streaming](/en/guides/server-side-rende
 
 #### `renderers` option
 
-**Type:** `AstroRenderer[]`
+**Type:** `SSRLoadedRenderer[]`
 
-A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script
+A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must use function `getContainerRenderer` that can be imported from the renderer library.
```

Comment:
```suggestion
A list of loaded client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) or MDX using an official Astro integration (e.g. React, Vue, etc.).

For each official Astro integration, use its `getContainerRenderer()` helper function that can be imported from package to expose its client and server rendering scripts. These are available for `@astrojs/react`, `@astrojs/preact`, `@astrojs/solid-js`, `@astrojs/svelte`, `@astrojs/vue`, `@astrojs/lit`, and `@astrojs/mdx`.

For renderer packages outside the `@astrojs` npm org, look in their documentation for `getContainerRenderer()` or a similar function provided.
```
Tried to incorporate the note here!

Also, I added "a list of LOADED client renderers" to further emphasize the `loadRenderers()` function use, and because the type has changed. I thought maybe this is nice for consistency? Up to you if you think that is helpful!